add-trusted-sha1-certs.sh

```diff
@@ -0,0 +1,50 @@
+# Helper script to facilitate generating two packages out of the certificate directory, one with
+# the certs as-is ($ORIG_SUFFIX) and one with all sha1 certs replaced by trusted certs ($TRUST_SUFFIX)
+
+# For each sha1-signed cert, create a duplicate trusted version to comply with EL9 default security policies
+# Give the original and trusted versions separate file suffixes, and also generate a separate set of
+# sha256sums for each set of certs
+
+# Directory to modify certs in
+CERT_DIR=$1
+# File suffix to apply to unmodified sha1 certs
+TRUST_SUFFIX=$2
+# File suffix to apply to modified sha1 certs
+ORIG_SUFFIX=$3
+
+# util function to find every sha1-signed cert
+find_sha1_certs() {
+    for f in $(find $1 -name "*.pem"); do 
+        if openssl x509 -noout -text < $f | grep "Signature Algorithm.*sha1" > /dev/null; then 
```

Comment:
```suggestion
        if openssl x509 -noout -text < $f | grep -q "Signature Algorithm.*sha1"; then 
```
-q makes grep just exit 0 immediately if a match was found which is what you want here.